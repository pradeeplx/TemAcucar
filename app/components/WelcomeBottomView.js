import React from 'react'
import { View, Platform, Dimensions } from 'react-native'
import Colors from "../Colors"

export default WelcomeBottomView = ({ children, style }) => {
  const dimensions = Dimensions.get('window')
  const height = dimensions.height * dimensions.scale
  const smallScreen = height <= 320
  return (
    <View style={[{
      position: 'absolute',
      bottom: (smallScreen ? 154 : (Platform.OS === 'ios' ? 32 : 60)),
      left: 0,
      right: 0,
      alignItems: 'center',
    }, style]}>
      { children }
    </View>
  )
}
