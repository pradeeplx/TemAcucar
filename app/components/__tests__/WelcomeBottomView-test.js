import 'react-native'
import React from 'react'
import WelcomeBottomView from '../WelcomeBottomView'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <WelcomeBottomView style={{top: 0}}>FooBar</WelcomeBottomView>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
