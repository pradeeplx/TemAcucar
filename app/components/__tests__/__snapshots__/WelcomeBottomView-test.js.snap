exports[`test renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "bottom": 32,
        "left": 0,
        "position": "absolute",
        "right": 0,
      },
      Object {
        "top": 0,
      },
    ]
  }>
  FooBar
</View>
`;
